Flow.Launcher.Core/Plugin/PluginsLoader.cs

```diff
@@ -46,8 +45,10 @@ public static IEnumerable<PluginPair> DotNetPlugins(List<PluginMetadata> source)
                         var assembly = assemblyLoader.LoadAssemblyAndDependencies();
                         var type = assemblyLoader.FromAssemblyGetTypeOfInterface(assembly,
                             typeof(IAsyncPlugin));
-
+#pragma warning disable IDE0019 // "Use Pattern Matching" - disabled because the pattern is useful for RELEASE
                         var plugin = Activator.CreateInstance(type) as IAsyncPlugin;
```

Comment:
why not use pattern matching?
like
```
if (Activator.CreateInstance(type) is not IAsyncPlugin plugin)
{
   ....
}
```

Reply:
It conflicts with the `RELEASE` and `DEBUG` code. Let me see if I can clean it up a bit.